WORKSPACE

```diff
@@ -1,12 +1,12 @@
 workspace(name = "com_github_bazelbuild_buildtools")
 
+# 0.5.5
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "1e8e662ab93eca94beb6c690b8fd41347835e8ce0f3c4f71708af4b6673dd171",
```

Comment:
this is required for security, I wouldn't remove it?
The problem here is that http://mirror.bazel.build/github.com/bazelbuild/rules_go/archive/2e319588571f20fdaaf83058b690abd32f596e89.tar.gz (or any URL of that form) isn't immutable and is generated on the fly, and has a different has each time.

See bazelbuild/bazel#3722

Reply:
Right, so how would sha256 work in concert with the mirror URL?

Reply:
It doesn't - according to the issue I linked to, we should use Bazel releases (which are created by the Bazel team, and are immutable).